scranShipper/src/DisplayFees/Index.jsx

```diff
@@ -0,0 +1,18 @@
+const DisplayFees = (props) => {
+    if (props.feeType === 'Total:') {
+    return (
+            <div className='d-flex justify-content-between  py-1'>
+                <p className='fw-bold'>{props.feeType}</p>
+                <p className='fw-bold'>{props.feePrice}</p>
+            </div>
```

Comment:
You don't need an if statement to return the whole thing, just set the classname for those two p tags based on the feetype